src/common/cachedTypes.js

```diff
@@ -79,9 +79,17 @@ Zotero.Connector_Types = new function() {
 			
 			this.getImageSrc = function(idOrName) {
 				var itemType = Zotero.Connector_Types["itemTypes"][idOrName];
-				var icon = itemType ? itemType[6]/* icon */ : "treeitem-"+idOrName+".png";
+				var icon = itemType ? itemType[6]/* icon */ : idOrName + '.svg';
+				icon = icon.replace('@2x', '');
```

Comment:
Can you add a comment to explain why we're stripping `@2x` here?

Reply:
Hmm. The answer is that I'm only processing the non @2x svgs in the build script, but the cached data only uses the @2x versions. But I don't know why. I don't even really understand why there are @2x svg icons. I thought vector graphics were dimensionless. @yexingsha ?

Reply:
> I don't even really understand why there are @2x svg icons. I thought vector graphics were dimensionless.

I had the same question, but after inspecting some, I believe these are artisanally crafted SVGs, with some optimizations for low-res displays. (E.g., the top-right paper fold was larger in the 1x version for one icon I looked at.)

Reply:
Right. I can add a comment if you have good suggestions for what it should be. I could see if we can use 2x svgs for hidpi displays, but even this is a big improvement over previous icons which were just pngs that were stretched, so no reason to hold this up.

Reply:
My PR (https://github.com/adomasven/zotero-connectors/pull/1) also aims to introduce the dark mode icons so this point can also be reexamined then!

Reply:
But is there a reason to only process the 1x SVGs? Are we not able to use the 2x ones? If anything, I'd favor optimizing for 2x at this point if that's an option.